### DISCUSSION TITLE
Air Quality sensor

### DISCUSSION THREAD

**Discussion by jkotowicz1** [joeycastillo/Sensor-Watch#215] [Ideas]
I wonder if anybody thought about adding air quality sensor to Sensor Watch? No idea if its feasible, but after a day in not so well ventilated office I wondered if one can measure and monitor CO2 levels from the watch.

**Comment by Vybo** [comment]
I don't think that would be possible, since the smallest CO2 sensor currently available is probably SCD4x and that itself is 7mm high, not even mentioning the other dimensions.

**Reply by potel123** [reply]
What about the BME680 ? It is tiny and measures Bosh Air Quality Index and can even differentiate smells :) Only donwside would be you need ventilation which istn compatible with waterproofness and also somewhat high power consumption.. Been also thinking about measureing indoor air quality!

**Reply by Vybo** [reply]
I don't think that one measures CO2 content in the air.

**Reply by potel123** [reply]
not directly but a proprietary air quality index from bosh which also measures VOCs which would be perfect for monitoring indoor air quiality...

**Comment by matheusmoreira** [comment]
A carbon monoxide sensor would be incredibly useful...
